cs/src/Connections/TunnelService.Connections.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'd suggest naming the `.csproj` files also: `TunnelService.Connections.csproj` -> `DevTunnels.Connections.csproj`, and similar.

Or we can wait and do further renaming in a separate change.

Reply:
Updated.